src/test/java/com/integrivideo/stories/ManageComponentStoryTest.java

```diff
@@ -50,6 +51,21 @@ public void componentCanBeUpdated() {
         componentSteps.componentDetailsShouldBeLike(ComponentTypeEnum.CLOUD_VIDEO_RECORDER, "edited");
     }
 
-    //TODO check prices for each component
-
+    @Test
```

Comment:
I suggest to make 3 different tests. One of the principle of good tests is 1 test - 1 check (execution time is not a big deal for now)